drivers/SmartThings/matter-window-covering/src/test/test_matter_window_covering.lua

```diff
@@ -129,7 +129,21 @@ local CLUSTER_SUBSCRIBE_LIST_NO_BATTERY = {
   WindowCovering.server.attributes.OperationalStatus,
 }
 
+local function set_preset(device)
+  test.socket.capability:__expect_send(
+    device:generate_test_message(
+      "main", capabilities.windowShadePreset.supportedCommands({"presetPosition", "setPresetPosition"}, {visibility = {displayed = false}})
+    )
+  )
+  test.socket.capability:__expect_send(
+    device:generate_test_message(
+      "main", capabilities.windowShadePreset.position(30, {visibility = {displayed = false}})
+    )
+  )
+end
```

Comment:
I'd maybe extend this test just to verify that the updated value is used for the next `presetPosition` command